api/seismic_site/settings.py

```diff
@@ -124,7 +125,8 @@ class Base(Configuration):
     # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
     STATIC_URL = "/static/"
-    STATIC_ROOT = os.path.join(BASE_DIR, "./public/static")
+    # STATIC_ROOT = os.path.join(BASE_DIR, "./public/static")
```

Comment:
The line should simply be removed, not commented 
Also, if you're here, can you remove the `STATICFILES_DIRS` from lines 130-132? 😅

Reply:
I was trying stuff to make sure it works and forgot to clean up. My bad.